src/pages/api/feedback.ts

```diff
@@ -0,0 +1,100 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { google } from "googleapis";
+import requestIp from "request-ip";
+import { UAParser } from "ua-parser-js";
+
+export default async function feedbackFormHandler(
+  request: NextApiRequest,
+  response: NextApiResponse
+) {
+  const { method } = request;
+  if (method !== "POST") {
+    return response
+      .status(405)
+      .send({ message: "only POST requests are allowed" });
+  }
+
+  // Values to pass to spreadsheet
+  const body = request.body;
+  const type = body.category;
+  const feedbackText = body.comment;
+  // timestamp: date
+  const currentDate = new Date();
+  const currentDayOfMonth = currentDate.getDate();
+  const currentMonth = currentDate.getMonth(); // Be careful! January is 0, not 1
```

Comment:
Is this the reverse in ruby?

Reply:
what do you mean? like month|date|year?

Reply:
I was asking if January is indexed 1 in ruby's date object.

Reply:
Ruby has a Time object that does the formatting out of the box https://ruby-doc.org/core-2.6.3/Time.html